regression/goto-analyzer/constant_propagation_02/test.desc

```diff
@@ -3,7 +3,7 @@ main.c
 --constants --simplify out.gb
 ^EXIT=0$
 ^SIGNAL=0$
-^Simplified:  assert: 1, assume: 0, goto: 1, assigns: 7, function calls: 0$
+^Simplified:  assert: 1, assume: 0, goto: 1, assigns: 8, function calls: 0$
```

Comment:
:pick: Please group these changes into a separate commit with why these numbers are changing

Reply:
In separate commit now.